.github/workflows/test.yml

```diff
@@ -21,6 +21,7 @@ jobs:
       - name: Install Python dependencies
         run: |
           pip install .[testing]
+          pip install pytest
```

Comment:
Now that pytest is in the `testing` extra requirement, this line is redudant btw. But doesn't hurt either.